addons/account_peppol/wizard/account_move_send.py

```diff
@@ -47,6 +47,12 @@ def _compute_checkbox_send_peppol(self):
         for wizard in self:
             wizard.checkbox_send_peppol = wizard.enable_peppol and not wizard.peppol_warning
 
+    @api.depends('enable_peppol')
+    def _compute_checkbox_send_mail(self):
+        super()._compute_checkbox_send_mail()
+        for wizard in self:
+            wizard.checkbox_send_mail = not (wizard.enable_peppol and not wizard.peppol_warning)
```

Comment:
```suggestion
            if wizard.checkbox_send_mail and wizard.checkbox_send_peppol:
                wizard.checkbox_send_mail = False
```
1. If you do that your `super()` call is totally useless as you will rewrite all records' values.
2. You're only recomputing `checkbox_send_peppol` value, this is obviously bad practice to duplicate code, just use the value directly.

Reply:
1- Totally agree
2- I did that because the dependencies should be `enable_peppol` not `checkbox_send_peppol`